src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1113,6 +1113,7 @@ describe('lib/viewers/BaseViewer', () => {
     describe('createAnnotator()', () => {
         const annotatorMock = {};
         const annotationsOptions = {
+            initialMode: 'none',
```

Comment:
Should these modes use the enum?

Reply:
Yeah I realize this when addressing comments in another PR lol